### ISSUE TITLE
Propositions de maquettes desktop + mobile v2.0

### ISSUE THREAD

**Issue by antoinepeltier** [anisayari/createurstech.fr#63]
Hello 👋

Une première sur GitHub pour moi donc soyez indulgents 🙂

J'ai vu passer un tweet, un design v1.0, des retours/remarques et voici une proposition pour une v2.0 avec un retravail de l'UX et de la hiérarchie des informations. J'ai aussi retravaillé l'Ui en restant dans les codes déjà posés.

[Le lien vers les fichiers Figma](https://www.figma.com/file/P2VUE8yiS40fyRbIa7Z6oU/Createurs-Tech?node-id=0%3A1) pour avoir les sources.
Le lien vers [la maquette desktop](https://www.figma.com/proto/P2VUE8yiS40fyRbIa7Z6oU/Createurs-Tech?page-id=0%3A1&node-id=1%3A2&viewport=472%2C276%2C0.25&scaling=min-zoom) et [la maquettes mobile](https://www.figma.com/proto/P2VUE8yiS40fyRbIa7Z6oU/Createurs-Tech?page-id=0%3A1&node-id=1%3A1203&viewport=472%2C276%2C0.25&scaling=min-zoom).

À dispo pour discuter des choix en design !
Ciao ! 😁

**Comment by DimitriTimoz** [comment]
Pas mal, le header + intro  sont magnifique, par contre je trouve la partie des filtres trop chargée sur l'écran pour la version mobile.
Sinon les boutons des filtres sur desktop sont pas mal aussi.

**Comment by antoinepeltier** [comment]
Merci :) Je vais voir pour modifier les paddings des filtres sur mobile et gagner en place et en légèreté.

**Comment by anisayari** [comment]
Pareil, le header est vraiment cool. Par. contre j'avou très être fan de la double colonne sur mobile. Et je pense que sur desktop il faut aussi plus de colonnes. Sinon le scroll sur plus de. 100 créateurs sera infini

**Comment by anisayari** [comment]
@DimitriTimoz l'idée du header avec le github est vraiment bien, tu penses pouvoir faire une PR avec ? ou je peux le faire. Mais si tu as du temps ca m'arrange.

**Comment by DimitriTimoz** [comment]
> @DimitriTimoz l'idée du header avec le github est vraiment bien, tu penses pouvoir faire une PR avec ? ou je peux le faire. Mais si tu as du temps ca m'arrange. 

Pas de problème je vais ça demain aprem si je peux car Orange à fait l'inverse d'un dépannage sur ma ligne donc j aurais que 5go demain. En espérant que ca suffise.

**Comment by DimitriTimoz** [comment]
Je ne peux pas aujourd'hui, je m en occupe demain si personne ne s en est occupé.

**Comment by anisayari** [comment]
> Je ne peux pas aujourd'hui, je m en occupe demain si personne ne s en est occupé.

Tu t'en es occupé ? Sinon je peux le faire.

**Comment by DimitriTimoz** [comment]
> > Je ne peux pas aujourd'hui, je m en occupe demain si personne ne s en est occupé.
> 
> Tu t'en es occupé ? Sinon je peux le faire.

Non, désolé, je n'ai trop le temps cette semaine en plus de mes problèmes d'internet. 
Je propose à mon cousin, il peut sans doute aider.

**Comment by AlexysGromard** [comment]
@anisayari Salut ! Du coup je m'occupe du header.

**Comment by anisayari** [comment]
> @anisayari Salut ! Du coup je m'occupe du header.

Top go ! Hésite pas à PR sur la branche test quand c'est ok.
